halo/hooks/object.cpp

```diff
@@ -28,9 +32,240 @@ Signature(true, sig_biped_jump,
     {0x8B, 0x0D, -1, -1, -1, 0x00,
      0x8B, 0x51, 0x34, 0x83, 0xEC, 0x10, 0x53, 0x55}); // size 6 do manual
 
+#define OBJECT_BEHAVIOR_FUNCTION_PTRS(type) \
```

Comment:
Minor stylistic nitpick: `OBJECT_BEHAVIOR_FUNCTION_PTRS` is a noun, but this name is being used for a function, so it should be a verb. What does this function actually do? Would something like `DECLARE_OBJECT_BEHAVIOR_FUNCTION_PTRS` make more sense?